assets/panel/debugger.properties

```diff
@@ -626,6 +632,10 @@ welcome.findInFiles=%S to find in files
 # panel. %S is replaced by the keyboard shortcut to search for functions.
 welcome.searchFunction=%S to search for functions in file
 
+# LOCALIZATION NOTE (welcome.allShortcuts): The label to open the modal of 
+# shortcuts, displayed in the welcome panel. 
+welcome.allShortcuts=Show all Shortcuts
```

Comment:
Doubt about capitalization here: should it be using title case? I'm not sure shortcuts should be uppercase on itself, but note that, in case, this is a product decision ;-)

Reply:
This is amazing @emanueltimlopez !  If you could update the text to "Show all shortcuts" I can merge it!  Thank you!

Reply:
Done!